File: src/main/java/cz/muni/fi/pa039/widgets/LineWidget.java
package cz.muni.fi.pa039.widgets;

import cz.muni.fi.pa039.Point;

public class LineWidget extends AbstractWidget {

    Point first;
    Point second;

    public LineWidget(Point first, Point second) {
        this.first = first;
        this.second = second;
    }
    @Override
    public boolean pointIsInside(int x, int y) {
        return false;
    }

    @Override
    public int getId() {
        return -2;
    }
}

File: src/cz/muni/fi/pa039/widgets/LineWidget.kt
package cz.muni.fi.pa039.widgets

import cz.muni.fi.pa039.Point
import java.awt.Color

class LineWidget(var first: Point, var second: Point) : AbstractWidget {
    override var isSelected = false
    override val id = -2
    override val fill = Color.RED.rgb

    override fun pointIsInside(x: Int, y: Int): Boolean {
        return false
    }
}
